templates/dashboard.html

```diff
@@ -9,37 +9,60 @@ <h1>Dashboard</h1>
     <div class="row">
         <div class="col-12">
             <div class="card">
-                <form action="/sony_ci/sync" method="POST">
+                <form action="/api/sony_ci/sync" method="POST">
                     <div class="card-header">
                         <div class="container-fluid">
                             <div class="d-flex justify-content-between align-items-center">
-                                <h3 class="card-title">Sony Ci</h3>
+                                <h3 class="card-title">Sync History</h3>
                             </div>
                         </div>
                     </div>
                     <div class="card-body border-bottom py-3">
                         <div class="mb-3">
-                            Last Sync was {{ sony_ci_last_sync.strftime('%a, %B %d at I:%m:%S %p') }}
-                        </div>
-                    </div>
-                    <div class="card-footer text-black">
-                        <div class="btn-list ms-auto justify-content-end">
-                            <button type="submit" name="_add_another" class="btn">
-                                <i class="fa-solid fa-rotate me-2">
-                                </i>
-                                {{ _("Full Sync")}}
-                            </button>
+                            <div class="table-responsive">
+                                <table class="table table-vcenter">
+                                    <thead>
+                                        <tr>
+                                            <th>Date / Time</th>
+                                            <th>Status</th>
+                                        </tr>
+                                    </thead>
+                                    <tbody>
+                                        {% for sync in sync_history %}
+                                        <tr>
+                                            <td>
+                                                {{ sync['created_at'].strftime('%Y-%m-%d %I:%M:%S %p') }}
```

Comment:
Can we change this to an iso8601 string for the frontend to parse?

```py
sony_ci_last_sync.isoformat()
```

Reply:
Can you elaborate on what you're looking for here?
```
sync['created_at'].strftime('%Y-%m-%d %I:%M:%S %p')
=> 2023-07-18 11:53:31 AM

sync['created_at'].isoformat()
=> 2023-07-18T11:53:31.965000

sync['created_at']
=> 2023-07-18 11:53:31.965000

```
The first one is more user friendly, and this template is the last stop before the front end display.